lib/mutant/mutator/node/begin.rb

```diff
@@ -13,23 +13,11 @@ class Begin < self
         #
         # @return [undefined]
         def dispatch
-          Util::Array.each(children, self, &method(:emit_child_subset))
           children.each_with_index do |child, index|
             mutate_child(index)
-            emit(child) unless children.eql?([child])
+            emit(child) unless children.one?
           end
         end
```

Comment:
I do not see this as required. But we should sync up on that one once. This code base is inconsistend in itself on that empty line. Lets avoid a cycle here for a global fix + enforcement to reduce that entropy in the future.